ethereum/env/.env.berachain.testnet

```diff
@@ -2,19 +2,20 @@
 # ethereum$ ln -s env/.env.berachain.testnet .env
 
 # Common config for forge deployment
-RPC_URL=https://artio.rpc.berachain.com/
+RPC_URL=https://bartio.rpc.berachain.com/
+FORGE_ARGS="--legacy"
 
 # Wormhole Core Migrations
 INIT_SIGNERS=["0x13947Bd48b18E53fdAeEe77F3473391aC727C638"]
 INIT_CHAIN_ID=39
 INIT_GOV_CHAIN_ID=0x1
 INIT_GOV_CONTRACT=0x0000000000000000000000000000000000000000000000000000000000000004
-INIT_EVM_CHAIN_ID=80085
+INIT_EVM_CHAIN_ID=80084
 
 # Bridge Migrations
 BRIDGE_INIT_CHAIN_ID=39
 BRIDGE_INIT_GOV_CHAIN_ID=0x1
 BRIDGE_INIT_GOV_CONTRACT=0x0000000000000000000000000000000000000000000000000000000000000004
-# Wrapped BERA: https://docs.berachain.com/developers/precompile-addresses#deployment-addresses
-BRIDGE_INIT_WETH=0x5806E416dA447b267cEA759358cF22Cc41FAE80F
+# Wrapped BERA:
```

Comment:
Worth keeping a link to https://docs.berachain.com/developers/deployed-contracts#deployment-contract-addresses ?